docs/index.rst

```diff
@@ -163,12 +163,6 @@ blobstorage_dir
 blobstorage_layout
   string
 
-Misc
-++++
-
-demostorage
-  boolean (if true, wrap FileStorage in a DemoStorage)
-
```

Comment:
Rather than removing the docs, please flag this option as deprecated (likewise below).

Reply:
Thanks, @tseaver. I restored documentation and added corresponding deprecation notice to it in https://github.com/Pylons/zodburi/pull/33/commits/3298ac19.

Glad to see you around!